reorder-journal.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+if ! [ -x "$(command -v hledger)" ]; then
+  echo 'Error: hledger is not installed.' >&2
+  exit 1
+fi
+
+SEPARATOR=";;;;;;;;;;;;;;;;;;;;;;;;;;;;;"
+LEDGER_FILE="$1"
+
+if [[ -z "$LEDGER_FILE" ]]; then
+  echo "Please specify the ledger file in the first argument. ./reorder-journal.sh hledger.journal"
+  exit 1
+fi
+
+TMPFILE_SORTED=$(mktemp /tmp/hledger.XXXXXX)
+TMPFILE_SKELETON=$(mktemp /tmp/hledger.XXXXXX)
+
+if ! hledger print -f "$LEDGER_FILE" > "$TMPFILE_SORTED";
+then
+  rm "$TMPFILE_SORTED" "$TMPFILE_SKELETON"
+  echo "The ledger file might be corrupted or incompatible with the current version of hledger."
+  exit 1
+fi
+
+if [[ 1 -ne $(fgrep "$SEPARATOR" "$LEDGER_FILE" | wc -l) ]];
+then
+  echo "The ledger file might be corrupted or incompatible with this script."
```

Comment:
How about something like:

```suggestion
  echo "The expected separator [show-separator] is missing or appears more than once."
```